app/controllers/v1/advice/decisions_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
     # scope to current user
     @person = Person.find_by!(external_identifier: params[:person_id])
     # needs options for order and eager loading (messages, events and records)
-    @decisions = @person.decisions.includes(:documents, :stakeholders, :messages, :events, :records).order("updated_at DESC").all
```

Comment:
i don't see `messages` or `events` being used in the decision serializer.

Reply:
okay, this is an old prototype we've archived so let's jsut remove it